CMakeLists.txt

```diff
@@ -9,10 +9,6 @@ project(icubWalkingDCM)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
```

Comment:
Maybe we should have a project name equal to the repo name.

(I cannot review in aline higher than this :sweat_smile:

Reply:
https://github.com/GiulioRomualdi/capture-point-walking/blob/8581d8df854ed390e205f44b1b4d4030dc93352a/CMakeLists.txt#L7

Reply:
Also related to start of the file: `CMAKE_CXX_STANDARD` is only available since CMake 3.1, so the CMake minimum required version should reflect  that. Furthermore, we are going to use it mostly using CMake 3.5 , so I propose to increase it to 3.5 .

Reply:
@S-Dafarra 
> Maybe we should have a project name equal to the repo name.

@traversaro 
> I propose to increase it to 3.5 .

I think now it's ok  https://github.com/GiulioRomualdi/capture-point-walking/blob/3dc913dfa0c00e84707b0d0f77c15615a2ef66cb/CMakeLists.txt#L5-L8